modules/fts/pages/fts-advanced-default-date-time-parser.adoc

```diff
@@ -0,0 +1,4 @@
+= Default Date/Time Parser
+The default date/time parser to be used for this bucket.
```

Comment:
Again not for the bucket -

The default datetime parser is the one applicable to all the datetime fields across the type mappings unless explicitly overridden